native_client/dotnet/DeepSpeechConsole/Program.cs

```diff
@@ -20,20 +21,34 @@ class Program
         static string GetArgument(IEnumerable<string> args, string option)
         => args.SkipWhile(i => i != option).Skip(1).Take(1).FirstOrDefault();
 
+        static string metadataToString(Metadata m) {
+            string retval = "";
+            Console.WriteLine($"metadataToString() - num_items={m.num_items}");
+            for (int i = 0; i < m.num_items; ++i) {
+                Console.WriteLine($"metadataToString() - num_items={m.num_items} - i={i}");
```

Comment:
@lissyx Why are you using metadataToString() inside the writeline?

Reply:
Well that's not what I meant, it was supposed to just be some debug printf, but I get what you mean :).

Reply:
@carlfm01 Even with that fixed, I see none of my debug printf :)